### ISSUE TITLE
Improve Readme

### ISSUE THREAD

**Issue by Nitesh-thapliyal** [Bauddhik-Geeks/Diabetes-Predictor#3]
Improve the readme of the repository by adding the requirements etc

**Comment by stutimongia2024** [comment]
hey @Nitesh-thapliyal , I would like to work on this issue. Please assign me the same.

**Comment by BassCoder2808** [comment]
Hi @Nitesh-thapliyal if the issue is still available, can you assign it to me?

**Comment by Nitesh-thapliyal** [comment]
@BassCoder2808 yes you can, you can improve how to contribute section. Assigning this issue to you

**Comment by galahad42** [comment]
@Nitesh-thapliyal is the issue still available. I can improve further on the contribution section to make it more descriptive. Or add a CONTRIBUTING.md
